packages/lib/modules/vebal/vebal-navigation.ts

```diff
@@ -0,0 +1,31 @@
+import { useSearchParams } from 'next/navigation'
+
+export const fromManage = 'manage'
+export const fromVote = 'vote'
+
+// Used to determine the redirect path after a vebal lock action
+type VeBalSourcePage = 'manage' | 'vote'
+
+export function getVeBalManagePath(
+  veBalAction: 'lock' | 'unlock' | 'extend',
+  sourcePage: VeBalSourcePage
+) {
+  return `/vebal/manage/${veBalAction}?from=${sourcePage}`
+}
+
+export function useVeBalRedirectPath() {
+  const searchParams = useSearchParams()
```

Comment:
do you know why Next.js makes this a hook even though the search params are never going to change after navigation (they could always be read from the URL)?